<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsPanel</class>
 <widget class="QWidget" name="NewsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="firstColumn">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="secondColumn"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="thirdColumn"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
